tests/cases/semantics/pats/multiple_spread.stderr

```diff
@@ -2,12 +2,12 @@ error: spread patterns `...` can only be used once in a list pattern
  --> $DIR/multiple_spread.hash:4:15
 3 |   main := () => {
 4 |       [...x, t, ...] := a
-  |                 ^^^ here
+  |                 ^^^ 
```

Comment:
Maybe add a note?

Reply:
what would the note specify?

Reply:
`cannot specify another spread pattern here` ?